packages/sdk/src/oracle/services/update.ts

```diff
@@ -159,5 +159,5 @@ export class Update {
     this.write((write) => {
       write.chains(chainId).optimisticOracle().request(update);
     });
-  }
+  };
```

Comment:
all these calls were refactored so we could pass them safely into other functions without worrying about `this`